plugin/Hydrogens.py

```diff
@@ -214,6 +214,9 @@ def match_and_update(self, atom_by_position, result_complex, polar=False):
                     if bond.atom2.symbol == 'H':
                         bond.atom2.polar_hydrogen = True
 
+        if unknown_atom_warnings:
+            Logs.warning('\n'.join(unknown_atom_warnings))
```

Comment:
hmm, have you tried sending this to graylog? I'm curious how it handles a potentially 4000+ line log message. Maybe we consolidate it to say "X Unknown atoms found" or "X hydrogen bonds failed to load". Or at most, we log the list of hydrogens that failed to load bonds, so its 4000 numbers, instead of 4000 lines

Reply:
good point, I have not tried sending it as I can't repro the issue where seemingly all the atoms fail to be found. I'll change the log to something shorter